public/pages/Dashboard/containers/Dashboard.js

```diff
@@ -459,7 +475,7 @@ export default class Dashboard extends Component {
     };
 
     const getItemId = (item) => {
-      if (perAlertView && isBucketMonitor) return item.id;
+      if (perAlertView) return isBucketMonitor ? item.id : `${item.id}-${item.version}`;
```

Comment:
Just to confirm, since we're using `${item.id}-${item.version}` so that selected items are updated upon acknowledgement and we don't do that when `isBucketMonitor`, does that mean the selected items won't be updated if the user acknowledged from the Monitor details page?

I know the flyout doesn't update without refresh for now, just wondering if that applies to the Monitor details table as well.

Reply:
It does update on the monitor details page. I think the difference is whether we add another call of getAlerts after acknowledging. Right here adding the `version` to item id is to support selecting single alert items, since they may share the same id but different version.

Reply:
Got it, since I believe the Monitor details page shows alerts as "per alert view" for Bucket-level Monitors as well, it could mean that acknowledging Alerts one by one there won't update them since we only use `item.id` in that scenario.

I think it's fine for now but if that is the case, let's capture it in the same issue we'll be capturing the flyout not updating on acknowledgement.

Reply:
No problem, thanks!